package.json

```diff
@@ -34,18 +37,21 @@
   },
   "homepage": "https://github.com/dojot/cron#readme",
   "dependencies": {
-    "@dojot/dojot-module": "^0.1.1",
-    "@dojot/dojot-module-logger": "^0.1.0",
-    "@dojot/healthcheck": "^0.1.0",
-    "axios": "^0.18.0",
-    "cron": "^1.6.0",
-    "cron-parser": "^2.7.3",
-    "express": "^4.16.4",
-    "express-validator": "^5.3.0",
-    "uuid": "^3.3.2",
-    "mongodb": "^3.1.13"
+    "@dojot/microservice-sdk": "^0.1.11",
```

Comment:
There is already a newer version of the SDK, check there, I believe there is no compatibility problem in updating it.

Reply:
This should be made.